src/main/java/software/amazon/qldb/QldbDriverImpl.java

```diff
@@ -247,7 +249,11 @@ private void retrySleep(RetryPolicyContext context, RetryPolicy retryPolicy) {
                 backoffDelay = Duration.ofMillis(0);
             }
 
-            TimeUnit.MILLISECONDS.sleep(backoffDelay.toMillis());
+            synchronized (this) {
```

Comment:
Revert this since sleep is fine in this case

Reply:
Reverted back to sleep.